### PR TITLE
Fix design issue with long application name, refs #586

### PR DESCRIPTION
**Pull request by poupotte**

### CODE REVIEW

File: client/app/views/home_application.coffee [hunk 1]
```diff
@@ -5,7 +5,7 @@ Modal = require './error_modal'
 
 # Row displaying application name and attributes
 module.exports = class ApplicationRow extends BaseView
-    className: "application w360-33 w640-25 full-20 mod left"
+    className: "application w360-33 w640-25 full-20 mod"
```
Comment (nono) [inline client/app/views/home_application.coffee]: @poupotte can you look at https://github.com/cozy/cozy-home/pull/600/files and see if there's a conflict?

Comment (poupotte) [inline client/app/views/home_application.coffee]: @lemelon , what do you think ? (You have more skill than me).

Comment (frankrousseau) [inline client/app/views/home_application.coffee]: It's a change made by @kelukelu

**Comment by clochix** [comment]
What is missing for merging this one ? Should we ping @kelukelu ?

**Pull request merged**